operators/click/__init__.py

```diff
@@ -90,8 +91,13 @@ def scrape_click():
     try:
         docs = json.load(open('click-cache.json'))
     except:
-        docs = requests.get('https://clickrevaha-sys.molsa.gov.il/api/solr?rows=1000').json().get('response').get('docs')
         json.dump(docs, open('click-cache.json', 'w'))
+        docs = (
+            requests.get(settings.SCRAPE_ENTRYPOINT_CLICK)
+            .json()
+            .get("response")
```

Comment:
is this with the black autoformatting? it changed here to double quotes, but left single quotes in other places.